frontend/src/views/admin/matching/index.tsx

```diff
@@ -1,14 +1,153 @@
 import React from "react";
+
 import { ContentArea } from "../../../components/layout";
 import { useSelector } from "react-redux";
 import { activeSessionSelector, fetchPostings } from "../../../api/actions";
 import { useThunkDispatch } from "../../../libs/thunk-dispatch";
-import { GroupedOverview } from "./grouped-overview";
+import { round } from "../../../libs/utils";
+
```

Comment:
Way too much in the `index.tsx` file. `index.tsx` should basically be used to re-export things from other files.